### PR TITLE
Cleanup some flowissues

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] test pinentry
- [x] test android tab bar

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers just some stuff i worked on on the plane from cali

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by buoyad** [comment]
@chrisnojima can you merge master in? This is on `0.68.0`. I'm doing it locally to review.

**Comment by chrisnojima** [comment]
k

File: shared/pinentry/remote-proxy.desktop.js [hunk 1]
```diff
@@ -2,71 +2,50 @@
...
 
 // Actions are handled by remote-container
-const RemotePinentry = compose(
-  connect(pinentryMapStateToProps, () => ({})),
-  SyncBrowserWindow,
-  SyncProps,
-  // $FlowIssue gets confused
-  renderNothing
-)(null)
+const RemotePinentry = compose(dataToProps, SyncBrowserWindow, SyncProps, renderNothing)(NullComponent)
```
Comment (buoyad) [inline shared/pinentry/remote-proxy.desktop.js]: is the `renderNothing` required here?

Comment (chrisnojima) [reply shared/pinentry/remote-proxy.desktop.js]: nope, i'll remove those tnx

**Comment by chrisnojima** [comment]
@buoyad ptal

**Pull request merged**